### PR TITLE
feat: enable wallet connect

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

[DEV-11698](https://app.clickup.com/t/2645698/DEV-11698)
[DEV-11870](https://app.clickup.com/t/2645698/DEV-11870)
[DEV-11922](https://app.clickup.com/t/2645698/DEV-11922)
[DEV-11869](https://app.clickup.com/t/2645698/DEV-11869)
[DEV-11926](https://app.clickup.com/t/2645698/DEV-11926)

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-11530 Integrate WalletConnect in up.cloud](https://app.clickup.com/t/2645698/DEV-11530)
Task linked: [DEV-11698 Enable WalletConnect](https://app.clickup.com/t/2645698/DEV-11698)

**Review by doubleppereira — APPROVED** [review]

**Review by JeneaVranceanu — APPROVED** [review]

**Comment by Hugoo** [comment]
Task linked: [DEV-11870 After deeplink, we should go back to dapp on mobile phone](https://app.clickup.com/t/2645698/DEV-11870)

File: utils/externalUrls.ts [hunk 1]
```diff
@@ -146,3 +148,28 @@ export const universalSwapsAssetUrl = (address?: Address) => {
   assert(isAddress(address), 'Address is not a valid address')
   return `https://universalswaps.io/tokens/lukso/${address.toLowerCase()}`
 }
+
+/**
+ * URL to Mobile App deep link used by Wallet Connect
+ *
+ * @param data
+ * @returns
```
Comment (JeneaVranceanu) [inline utils/externalUrls.ts:156]: I'd remove this if you are not planning to write anything here.

**Review by JeneaVranceanu — APPROVED** [review]
LGTM

File: utils/externalUrls.ts (hunk 1 above)
Comment (dzbo) [reply utils/externalUrls.ts:156]: Makes sense

**Review by richtera — APPROVED** [review]

**Review by richtera — APPROVED** [review]

**Comment by Hugoo** [comment]
Task linked: [DEV-11869 On mobile phone, enable deeplink for transactions](https://app.clickup.com/t/2645698/DEV-11869)
Task linked: [DEV-11922 Investigation: WalletConnect connection is long](https://app.clickup.com/t/2645698/DEV-11922)

**Review by JeneaVranceanu — APPROVED** [review]

**Comment by Hugoo** [comment]
Task linked: [DEV-11926 Bug: on mobile phone, if we want to send asset to UP, the network switch modal appears](https://app.clickup.com/t/2645698/DEV-11926)

**Pull request merged**